Catalog tag filter (#5280)
SkylineCommunications/dataminer-docs: dataminer/dataminer_services/Catalog/Looking_up_a_catalog_item.md

   - **Public**: This will only return results that are publicly available to everyone.
   - **Private**: This will only return results from the private items that are only visible to people in your organization.

   > [!TIP]

>    > If you see a tag in the search results that interests you, click the tag to quickly filter the results to items with that tag only.

Author: ```suggestion
   > If you see a tag in the search results that interests you, click the tag to quickly filter the results to items that match that tag.
```
